site/en/docs/webstore/images/index.md

```diff
@@ -5,28 +5,33 @@ date: 2018-06-11
 description: Guidelines about the kinds of images you need to supply to the Chrome Web Store.
 ---
 
-{% Aside 'warning' %}
-**Important:** Chrome will be removing support for Chrome Apps on Windows, Mac, and Linux. Chrome OS
-will continue to support Chrome Apps. Additionally, Chrome and the Web Store will continue to
-support extensions on all platforms. [Read the announcement][1] and learn more about [migrating your
-app][2].
-{% endAside %}
-
 You need to supply several kinds of images to be used in the Chrome Web Store and the Google Chrome
 browser:
 
-- [App icon][3]
+- [Extension icon][3]
 - [Promotional images][4]
 - [Screenshots][5]
 
-Only the app icon, a small promotional image, and a screenshot are mandatory. However, providing
-attractive versions of both required and optional images increases your app's chances of getting
-noticed. For example, your app can't be featured unless you provide a large or marquee promotional
-image.
+Only the extension icon, a small promotional image, and a screenshot are mandatory. However, providing
+attractive versions of both required and optional images increases your extension's chances of getting
+noticed. For example, your extension can't be featured in marquee unless you provide a marquee promotional
+image. Follow the below image best practices to increase your chance of being featured.
```

Comment:
Not a fan of "the below &lt;noun&gt;". Consider something like...
"Follow the below image best practices" --> "Observe the following image best practices"

Or perhaps:
Follow our [image best practices] to increase...
.
.
.
[image best practices]: #image-best-practices

Reply:
done!

Reply:
Line 318 isn't right. The generated anchor for that heading is `image-best-practices`, not `imagebestpractices`.